.gitignore

```diff
@@ -1,5 +1,17 @@
 leelaz
 *.o
 *.d
+.vscode/settings.json
 training/tf/venv
-weights.txt
\ No newline at end of file
+.vs/
+packages/
+*.db
+*.opendb
+*.user
+*.orig
+src/x64
+x64/
+autogtp/*.sgf
```

Comment:
There's a .gitignore in the autogtp directory.

Reply:
This must came from some earlier version of .gitignore, and being kept during merge.